components/ResultsPage/BenefitCard.tsx

```diff
@@ -43,9 +43,12 @@ export const BenefitCard: React.VFC<{
 
   return (
     <div className="my-6 py-6 px-8 border border-[#6F6F6F] rounded">
-      <h2 id={benefitKey} className="h2 inline-flex justify-start items-center">
-        <span className="flex-1">{benefitName}</span>&nbsp;{eligibleFlag}
-      </h2>
+      <div className="inline">
+        <h2 id={benefitKey} className="inline align-sub h2">
+          {benefitName}
+        </h2>
+        {eligibleFlag}
+      </div>
```

Comment:
this was the original code a few weeks ago, when was it changed? 
```
    <div className="my-6 py-6 px-8 border border-[#6F6F6F] rounded">
      <h2 id={benefitKey} className="h2">
        {benefitName}&nbsp;{eligibleFlag}
      </h2>
```

Reply:
Yesterday. Since the badge and the text is not vertically aligned.

Reply:
what I mean the original code was ok.

Reply:
@MarcoGoC  You can switch back to it, and you will see that the badge is lower than the heading text, which is different from the mockup.